examples/camera-app/linux/include/webrtc-abstract.h

```diff
@@ -52,9 +52,10 @@ class WebRTCTrack
 public:
     virtual ~WebRTCTrack() = default;
 
-    virtual void SendData(const char * data, size_t size) = 0;
-    virtual bool IsReady()                                = 0;
-    virtual std::string GetType()                         = 0; // "video" or "audio"
+    virtual void SendData(const char * data, size_t size)                     = 0;
```

Comment:
drive by review comments:

Could we use Span/CharSpan for the "data + size" groups?
Is data really char * rather than uint8_t *?

Reply:
Thanks for your review. I believe libdatachannel expects std::byte*, but char* is used in many places right now in the camera app, like data extracted from gst pipeline, push-av and libdatachannel components. I think we can make use of ByteSpan in these places. I'll include this in the upcoming PR if you could agree. Always thank you for your time.